components/Toolbar/ToolbarLayout.js

```diff
@@ -15,18 +15,26 @@ class ToolbarLayout extends React.Component {
 
   render() {
     const { filters, children, filterClearValues } = this.props;
+    const { formatMessage } = this.props.intl;
+
     return (
       <div className="row toolbar-pf">
         <div className="col-sm-12">
           <div className="toolbar-pf-actions">{children}</div>
           {filters.filterValues && filters.filterValues.length !== 0 && (
             <Toolbar.Results>
-              <Filter.ActiveLabel>Active Filters:</Filter.ActiveLabel>
+              <Filter.ActiveLabel>
+                <FormattedMessage defaultMessage="Active Filters" />:
+              </Filter.ActiveLabel>
               <Filter.List>
                 {filters.filterValues.map(item => {
                   return (
                     <Filter.Item key={`${item.key}-${item.value}`} onRemove={this.handleRemoveFilter} filterData={item}>
-                      {item.key}: {item.value}
+                      {formatMessage({
+                        id: "filter key",
+                        defaultMessage: item.key
```

Comment:
My comments [above](https://github.com/osbuild/cockpit-composer/pull/972/files#r432127303) also apply to this case.

Reply:
Fixed